apps/trading/e2e/fixtures/market.py

```diff
@@ -8,12 +8,16 @@
 mint_amount: float = 10e5
 market_name = "BTC:DAI_2023"
 
+default_sell_orders = [[1, 110], [1, 105]]
+default_buy_orders = [[1, 90], [1, 95]]
+
 def setup_simple_market(
     vega: VegaService,
     approve_proposal=True,
     custom_market_name=market_name,
     custom_asset_name="tDAI",
     custom_asset_symbol="tDAI",
+    custom_quantum=1
```

Comment:
Not one for this PR but I think we should default to a more sensible quantum at some point. But it will likely break a lot of tests.

Reply:
sure, I just set this to match the default in the lower level method for now

Reply:
I also think a lot of these default params should be stored in a central .py file somewhere so we have a single source of truth